test/test_techcrunch.py

```diff
@@ -0,0 +1,39 @@
+from test.conftest import MockUQCSBot, TEST_CHANNEL_ID
+from unittest.mock import patch
+
+import feedparser
+from typing import List, Dict
+
+def get_local_techcrunch_data() -> List[Dict[str, str]]:
+    """
+    Returns the RSS data from local mock copy of techcrunch xml.
+    """
+    try:
+        rawdata = open("test/techcrunch.xml").read()
+        data = feedparser.parse(rawdata).entries
+    except Exception as e:
+        raise e
+        return (0, "")
```

Comment:
This is superfluous. return will never be reached and there's no need to catch then immediately re-raise an exception.

Reply:
So would you like me to print something before raising the exception?

Reply:
I'd just let the exception throw, but printing something is fine too.

Reply:
I think Kenton is just suggesting that you either a) don't catch it at all, or b) catch it, send a message indicating the error to Slack and then exit without re-raising. My preference would be for b).

Reply:
@nicklambourne this try block is in the test file though, so there's no point sending a message to Slack right? Or is there a specific channel to send test fail messages to?

Reply:
Generally, sending an error message is good but this is for testing so we can't do that here.